### PR TITLE
TELCODOCS:1040 - remove kata monitor image

### PR DESCRIPTION
**Pull request by MirzWeiss**
Version(s):
Main, 4.12, 4.13

Issue:
[TELCODOCS-1040](https://issues.redhat.com/browse/TELCODOCS-1040)

[Link to docs preview 1](https://54140--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads.html#sandboxed-containers-create-kataconfig-resource-web-console_deploying-sandboxed-containers)
[Link to docs preview 2](https://54140--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads.html#sandboxed-containers-create-kataconfig-rsource-cli_deploying-sandboxed-containers)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/sandboxed-containers-create-kata-config-resource-cli.adoc [hunk 1]
```diff
@@ -58,7 +60,8 @@ kind: KataConfig
 metadata:
   name: cluster-kataconfig
 spec:
-  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:{sandboxed-containers-version}.0
+  checkNodeEligibility: false <
```
Comment (cpmeadors) [inline modules/sandboxed-containers-create-kata-config-resource-cli.adoc:63]: the "<" looks like it doesn't belong

**Review by cpmeadors — APPROVED** [review]
With the exception of the one small comment, everything matches my understanding of how things work.  I think the kataconfig fields are better explained.

**Comment by MirzWeiss** [comment]
> With the exception of the one small comment, everything matches my understanding of how things work. I think the kataconfig fields are better explained.

@cpmeadors Thank you! And thank you so much for all of your help and support on this!

**Review by kcarmichael08 — COMMENTED** [review]
Overall this looks good - had a few comments and suggestions. Hope they are useful!

File: modules/sandboxed-containers-create-kata-config-resource-cli.adoc [hunk 2]
```diff
@@ -46,8 +46,10 @@ kind: KataConfig
 metadata:
   name: cluster-kataconfig
 spec:
-  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:{sandboxed-containers-version}.0
+  checkNodeEligibility: false <1>
+  logLevel: info
 ----
+<1> Set`checkNodeEligibility` to `true` to detect node eligibility to run `kata` as a `RuntimeClass`. For more information, see "Checking whether cluster nodes are eligible to run {sandboxed-containers-first}" above.
```
[the remark this answers is not in the document]
Comment (MirzWeiss) [reply modules/sandboxed-containers-create-kata-config-resource-cli.adoc:52]: Got it. Thanks!

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc [hunk 3]
```diff
@@ -38,36 +38,31 @@ Kata is installed on all worker nodes by default. If you want to install `kata`
 
 . In the *KataConfig* tab, click *Create KataConfig*.
 
-. In the *Create KataConfig* page, select to configure the `KataConfig` CR via *YAML view*.
+. In the *Create KataConfig* page, define the following fields:
```
[the remark this answers is not in the document]
Comment (MirzWeiss) [reply modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:41]: Just entering information in each of the fields. I'll clarify the language here.

**Comment by MirzWeiss** [comment]
/label merge-review-needed

Please review. However, if it's good to go, **do not merge** yet, please! This is slated for release on January 24th. I just need the merge review done so it can be merged on the release date. I'll request the actual merge separately.

**Pull request merged**